src/commands/protocols/publish/receiver/v1.0.0/v1-0-0-handle-store-init-command.js

```diff
@@ -1,3 +1,4 @@
+import { BigNumber } from 'bignumber.js';
```

Comment:
Is this intended? I am asking because in other part of the code we are using BigNumber from Ethers.js

Reply:
Yes, for float precision when calculating bid. BigNumber from ethers library doesn't support decimals